karma.conf.js

```diff
@@ -12,20 +10,28 @@ module.exports = function (config) {
     // Only test IE latest.
     config.browsers.push('internet_explorer_11');
   }
+  var polyOptions = process.argv.filter((val) => {
```

Comment:
Do you think it'd be worth bringing something in like [`minimist`](https://github.com/substack/minimist)?